datafusion/sql/src/statement.rs

```diff
@@ -709,25 +711,42 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             }
         };
 
-        // TODO, parse options as Vec<(String, String)> to avoid this conversion
-        let options = statement
-            .options
-            .iter()
-            .map(|(s, v)| (s.to_owned(), v.to_string()))
-            .collect::<Vec<(String, String)>>();
+        let mut options = HashMap::new();
```

Comment:
Now, we handle COPY TO statements here, by creating the format options for the COPY logical plan.